kafka/producer/producer.go

```diff
@@ -13,18 +13,18 @@ import (
 
 	"github.com/Shopify/sarama"
 	"github.com/aristanetworks/glog"
-	"github.com/golang/protobuf/proto"
+	pb "github.com/openconfig/gnmi/proto/gnmi"
 )
 
 // Producer forwards messages recvd on a channel to kafka.
 type Producer interface {
 	Start()
-	Write(proto.Message)
+	Write(*pb.SubscribeResponse)
 	Stop()
 }
 
 type producer struct {
-	notifsChan    chan proto.Message
+	notifsChan    chan *pb.SubscribeResponse
```

Comment:
This is failing tests, I'll fix it along with the other comments on my end. Thank you for your contribution